lib/Conversion/TritonGPUToLLVM/ConvertLayoutOpToLLVM/SharedToDotOperandMFMA.cpp

```diff
@@ -439,6 +445,9 @@ Value loadA(ConversionPatternRewriter &rewriter, Location loc, Value thread,
       std::max<int>(mfmaInstrM * mfmaInstrK / iWaveSize /*wave size*/, 1);
   unsigned int maxNumWarps = shape[0] / mfmaInstrM;
   int warpsPerGroupM = std::min(warpsPerCTA[0], maxNumWarps);
+  if (isF8(aElemTy)) {
+    aElemTy = i8_ty;
+  }
```

Comment:
Optional:
You can use TypeConverter for this:
```suggestion
    aElemTy = typeConverter->convertType(aElemTy);
```
Plus I want to refactor code on line 471 and 492, you can do it in this PR, if you want.

Reply:
Thanks. Changed all places.